eessi/testsuite/tests/apps/cp2k/input/QS/H2O-128.inp

```diff
@@ -0,0 +1,449 @@
+&GLOBAL
+  EPS_CHECK_DIAG 1.0E-12
+  PRINT_LEVEL LOW
+  PROJECT H2O-128
+  RUN_TYPE MD
+&END GLOBAL
+
+&MOTION
+  &MD
+    ENSEMBLE NVE
+    STEPS 10
+    TEMPERATURE 300.0
+    TIMESTEP 0.5
+  &END MD
+&END MOTION
+
+&FORCE_EVAL
+  METHOD QS
+  &DFT
+    BASIS_SET_FILE_NAME GTH_BASIS_SETS
+    POTENTIAL_FILE_NAME POTENTIAL
+    &MGRID
+      CUTOFF 280
+      REL_CUTOFF 30
+    &END MGRID
+    &QS
+      EPS_DEFAULT 1.0E-12
+      EXTRAPOLATION_ORDER 3
+      WF_INTERPOLATION PS
+    &END QS
+    &SCF
+      # IGNORE_CONVERGENCE_FAILURE
```

Comment:
make it clear why this is commented out?

```suggestion
      # IGNORE_CONVERGENCE_FAILURE  # only for CP2K >= 2024.1
```

`sed 's/### IGNORE_CONVERGENCE_FAILURE/ IGNORE_CONVERGENCE_FAILURE /g`